streaming_kafka_consumer/tests/backends/test_kafka.py

```diff
@@ -54,7 +52,9 @@ def test_payload_pickle_out_of_band() -> None:
 
 class KafkaStreamsTestCase(StreamsTestMixin[KafkaPayload], TestCase):
 
-    configuration = get_default_kafka_configuration()
+    configuration = build_kafka_configuration_with_overrides(
+        {"bootstrap.servers": os.environ.get("DEFAULT_BROKERS", "localhost:9092")}
+    )
```

Comment:
Why was this needed ?

Reply:
This test requires the actual Kafka connection (and we obviously don't want to depend on Snuba settings to get that). This will probably be tweaked (and moved to a central place) later when this moves to another repository as that environment variable may not be used anymore.